accepted/2021/experimental-package-shading.md

```diff
@@ -0,0 +1,59 @@
+# Experiment: NuGet Package Shading
+
+(intro and scenarios to be added)
+
+## Usage
+
+### Package authors
+
+A *package author* may enable shading on any package reference as long as its assets are configured such that it is effectively private. Any other package references that have a shaded package reference as a transitive dependency must also be shaded. When the project is restored, shaded copies of the packages are created and swapped in. When the project is packed, the shaded assets are embedded in the resulting package.
+
+### Package consumers
+
+Shading is superficially invisible to consumers of a package that has shaded dependencies. They may notice the shaded assets in the build logs or output directory, and their app size may increase, but they do not need to be aware that shading exists. From the consumer's perspective, a shaded dependency is no different than any other private asset.
+
+## Behavior
+
+### Private assets
+
+This shading mechanism is built upon the existing NuGet concept of *private assets*. A shaded asset is inherently private and must be treated the same way as other private assets. Handling shaded assets using the private assets mechanisms reuses existing behavior and concepts, and should result in a consistent experience.
```

Comment:
Can we link it to docs on Private assets?

Reply:
Added a link :)